### DISCUSSION TITLE
[Solved] Can't see my stats

### DISCUSSION THREAD

**Discussion by zorluozan** [cssbattle/forum#232] [Bugs reports]
When I try to enter my profile, it keeps running and says "Fetching player data" and not opening my profile. Here is my profile link;
https://cssbattle.dev/player/ozzy

**Comment by kushsolitary** [comment]
> When I try to enter my profile, it keeps running and says "Fetching player data" and not opening my profile. Here is my profile link; https://cssbattle.dev/player/ozzy

Hi @zorluozan, we've identified the issue and we'll fix it in the coming weeks. Thanks for reporting!

**Comment by chinchang** [comment]
We made some optimizations on the backend and this issue is now fixed, finally!
Let us know if anyone still faces this issue.

**Reply by zorluozan** [reply]
Yes it is fixed. Thanks a lot.
